examples/scripts/ct_astra_unet_train_foam2.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+CT with UNet for Denoising of FBP
+=================================
```

Comment:
Same comments as for `ct_astra_modl_train_foam2.py`.

Reply:
Keeping 'train' in the filename for compatibility with the CI workflow: train scripts should be skipped, the filename is used to identify such scripts.